projects/spectator/jest/test/set-input-alias-names.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from '@angular/core';
+import { createComponentFactory } from '@ngneat/spectator/jest';
+
+describe('SetInputAliasNames', () => {
+  @Component({
+    selector: 'app-root',
+    template: `
+      <div data-test="set-input--name">{{ name }}</div>
+      <div data-test="set-input--age">{{ numOfYears }}</div>
+    `,
+    standalone: true,
+  })
+  class DummyComponent {
+    @Input('userName') name!: string;
```

Comment:
Can you please also add the signal input alias variations?

Reply:
sure! I added the relevant unit tests